packages/angular/src/table/table.stories.ts

```diff
@@ -0,0 +1,82 @@
+import { boolean, select, withKnobs } from '@storybook/addon-knobs';
+import { action } from '@storybook/addon-actions';
+import { moduleMetadata, storiesOf } from '@storybook/angular';
+import { TableHeaderItem, TableItem } from 'carbon-components-angular';
+
+import { AITableModel } from './table-model.class';
+import { AITableModule } from './table.module';
+
+const simpleModel = new AITableModel();
+
+simpleModel.setHeader([
+  [
+    new TableHeaderItem({
+      data: 'Name',
+      rowSpan: 2,
+    }),
+    new TableHeaderItem({ data: 'hwer', colSpan: 2, sortable: false }),
+    null,
+  ],
+  [null, new TableHeaderItem({ data: 'hwer1' }), new TableHeaderItem({ data: 'hwer2' })],
+]);
+
+simpleModel.setData([
```

Comment:
The body content isn't showing for me in the story 🤔 Not sure if that's intentional since we're still pretty early or?